README.md

```diff
@@ -6,6 +6,16 @@
 
 happy-heron, or H2 (from "Hydrus 2.0"), is a Rails web application enabling users to deposit scholarly content into the SDR. It replaced [Hydrus](https://github.com/sul-dlss/hydrus).
 
+**********************************
+__H2 is currently frozen in production__
+
+* The `production` branch is what is currently deployed in production.
+* Any neccessary production patches should be deployed to this branch and deployed with `cap prod deploy`.
```

Comment:
s/deployed to this branch/applied to this branch/ ?

Reply:
Fixed.